### ISSUE TITLE
Fishing: Go through schools of fish

### ISSUE THREAD

**Issue by ccshiro** [ccshiro/cc-buglist#755]
From what I've been told the pools/schools in fishing tend to be clumped up on mangos and with a spawn rate that's too high. We should go through this.

**Comment by ccshiro** [comment]
An example, I was given:

"All of the fishing pools in Darkshore are Floating Debris, so it's almost cheating since you can get some good stuff from those."

**Comment by Lephidiles** [comment]
I think without research that it is like this (multiple spawns in one spot) that this should be what rot calls pooled... that there is one spawn point that can spawn one of say tree schools but not all 3 at once (should pick a random ? school to spawn out of the said 3)

This also apply to gold / silver ore afaik that they should be a proc spawn of a copper ore <-- again not sure this is entirely true (needs research i guess) 

basicly a lot of vanilla was like this (npc, (rare) ore, (rare) herbs, schools etc)

**Comment by Lephidiles** [comment]
http://www.wowhead.com/object=1733/silver-vein#comments
http://www.wowhead.com/object=1734/gold-vein#comments
etc.
(it was not copper though)

still only wowhead comment sources

**Comment by ccshiro** [comment]
@Lephidiles yeah, that's what I think too. We've already pooled mines, herbs and chests. I'm guessing schools of fish should be pooled as well.

**Comment by rsheep** [comment]
For sure, even schoois of fish were pooled.
Prime example for this was "Stranglethorn Fishing Extravaganza".

**Comment by Krack3n77** [comment]
Next test I'll record footage of how insane the pool and wreckage/etc spawns are. Leveled to 130 fishing last test. The docks in Ratchet were crazy.
